<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка тарифов из ИНФИС</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>импортировать из</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="8">
    <widget class="QLineEdit" name="edtFileName">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="gbDublicates">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Совпадающие записи</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="chkUpdate">
        <property name="text">
         <string>Обновить</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chkSkip">
        <property name="text">
         <string>Пропустить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chkAskUser">
        <property name="text">
         <string>Спросить у пользователя</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="11">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="11">
    <widget class="QTextBrowser" name="log"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="btnImport">
     <property name="text">
      <string>начать импортирование</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QGroupBox" name="gbPeriod">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Период</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblBegDate">
        <property name="text">
         <string>с</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CDateEdit" name="edtBegDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblEndDate">
        <property name="text">
         <string>по</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CDateEdit" name="edtEndDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="3" colspan="8">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="chkLoadMature">
       <property name="text">
        <string>Загружать взрослые</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLoadChildren">
       <property name="text">
        <string>Загружать детские</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLoadMes">
       <property name="text">
        <string>Загружать МЭС</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLoadService">
       <property name="text">
        <string>Загружать услуги</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLoadVisit">
       <property name="text">
        <string>Загружать посещения</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="10">
    <widget class="QPushButton" name="btnView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Просмотр</string>
     </property>
    </widget>
   </item>
   <item row="8" column="10">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>закрыть</string>
     </property>
    </widget>
   </item>
   <item row="8" column="8">
    <widget class="QLabel" name="labelNum">
     <property name="text">
      <string>всего записей в источнике:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QToolButton" name="btnSelectFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="11">
    <widget class="QGroupBox" name="gbRefBooks">
     <property name="title">
      <string>Справочники для определения типа тарификации</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblRbProfile">
        <property name="text">
         <string>Справочник профилей</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtRbProfileFileName"/>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="btnViewRbProfile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Просмотр</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="7">
       <widget class="QLabel" name="lblRbLoadedInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnLoadRbProfile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="btnSelectFileRbProfile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkUpdate</tabstop>
  <tabstop>chkSkip</tabstop>
  <tabstop>chkAskUser</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>